pkgs/tools/typesetting/sile/default.nix

```diff
@@ -46,11 +44,12 @@ stdenv.mkDerivation rec {
   enableParallelBuilding = true;
 
   checkPhase = ''
-    make documentation/developers.pdf documentation/sile.pdf
+    make documentation examples
   '';
 
   postInstall = ''
-    install -D -t $out/share/doc/sile documentation/*.pdf
+    install -D -t $out/share/doc/sile documentation/sile.pdf
```

Comment:
shouldn't upstream install the doc ? or provide a target `install-doc` ?

Reply:
Can you point me do documentation or a good case study for how `make install-doc` should work? I'd be happy to get it setup properly upstream but I'm unsure what the outcome should look like.

Reply:
The manual (documentation/sile.pdf) actually comes precompiled in the source tarballs now so you can skip the `make documentation` unless you are building from git HEAD. Note this is also why Gentium-Book and DeJavu fonts are not requirements. If building the manual you'd need those back.